.travis.yml

```diff
@@ -3,40 +3,49 @@ addons:
   apt_packages:
   - zsh
   - ksh
+
```

Comment:
let's revert the changes to this file - the need for spacing is subjective, and it's not necessary for this PR